qiskit_algorithms/optimizers/scipy_optimizer.py

```diff
@@ -116,9 +130,16 @@ def minimize(
         jac: Callable[[POINT], POINT] | None = None,
         bounds: list[tuple[float, float]] | None = None,
     ) -> OptimizerResult:
-        # Remove ignored parameters to suppress the warning of scipy.optimize.minimize
+
+        # Allocate bounds attribute
+        if bounds is not None:
+            self._bounds = bounds
```

Comment:
Same as the comment in __init__, since we don't run checks at the constructor level, we don't need to store it anymore. So I agree with you and removed this part too in the next commit.